### PR TITLE
Bug/helm ivp error report

### LINKED ISSUE #101: MOOS-IvP Health reporting bugs
- Disable reporting for MOOS-IvP health when running jaiabot_health on the hub.
- I'm seeing periodic reports of ERROR__MOOS__HELMIVP_STATE_NOT_DRIVE when everything is working OK.
   - I think this is due to the mismatch in resetting status_ and receiving the next data from MOOS.
   - I think the quick fix is to store the previous jaiabot::protobuf::HelmIVPStatus for use in the health() method, rather than the current.

### PR DESCRIPTION
**Pull request by ferrom-jaia**

### CODE REVIEW

File: src/bin/health/helm_ivp_thread.cpp [hunk 1]
```diff
@@ -73,9 +73,12 @@ jaiabot::apps::HelmIVPStatusThread::HelmIVPStatusThread(
 
 void jaiabot::apps::HelmIVPStatusThread::issue_status_summary()
 {
-    glog.is_debug2() && glog << group(thread_name()) << "Status: " << status_.DebugString()
+    glog.is_debug2() && glog << group(thread_name()) << "Status: " << previous_status_.DebugString()
```
Comment (tsaubergine) [inline src/bin/health/helm_ivp_thread.cpp]: I think you can safely publish status_ here but I'm not sure how much different it'll make (just delays error reporting by loop() interval).

**Comment by tsaubergine** [comment]
Closes #101

**Comment by tsaubergine** [comment]
While I think this improves the situation, it's still happening for me some in the simulator. I'd suggest we implement this functionality in `jaiabot::moos::IvPHelmTranslation` (jaiabot/src/lib/moos_gateway/jaiabot_gateway_plugin.h) by implementing the virtual method:

void health(goby::middleware::protobuf::ThreadHealth& health) override

I'm wondering if we're seeing a bit of lag reading all the moos variables (the idea behind the `jaiabot::groups::moos` group was mostly for logging purposes).

**Comment by ferrom-jaia** [comment]
> While I think this improves the situation, it's still happening for me some in the simulator. I'd suggest we implement this functionality in `jaiabot::moos::IvPHelmTranslation` (jaiabot/src/lib/moos_gateway/jaiabot_gateway_plugin.h) by implementing the virtual method:
> 
> void health(goby::middleware::protobuf::ThreadHealth& health) override
> 
> I'm wondering if we're seeing a bit of lag reading all the moos variables (the idea behind the `jaiabot::groups::moos` group was mostly for logging purposes).

https://jaia-innovation.atlassian.net/wiki/spaces/PD/pages/863961102/Videos

**Review by tsaubergine — APPROVED** [review]

**Pull request merged**